File: php/examples/post_auth_login.php
<?php
require_once 'base_request.php';

// Example of POST /auth/login

$debug = true;
$email = $_ENV['EMAIL'];
$password = $_ENV['PASSWORD'];
$website_id = $_ENV['WEBSITE_ID'];  // optional

$request_data = array(
    'email' => $email,
    'password' => $password,
    'website_id' => $website_id
);

$request_params = '';

$json_body = json_encode($request_data);

send_request('POST', '/auth/login', $request_params, $json_body);

File: csharp/examples/PostAuthLogin.cs
using System;
using System.Diagnostics;

namespace Examples.examples
{
    // Example of POST /auth/login
    
    public class PostAuthLogin : BaseRequest
    {
        private string _email;
        private string _password;
        private string _website_id;

        public PostAuthLogin()
		{
            _email = Environment.GetEnvironmentVariable("EMAIL");
            _password = Environment.GetEnvironmentVariable("PASSWORD");
            _website_id = Environment.GetEnvironmentVariable("WEBSITE_ID");

            Console.WriteLine($"PostAuthLogin: _email={_email} _password={_password} _website_id={_website_id}");
        }

        public override void SendRequest()
        {
            var body = Newtonsoft.Json.JsonConvert.SerializeObject(new
            {
                email = _email,
                password = _password,
                website_id = _website_id
            });

            PostRequest("auth/login", body);
        }
    }
}